applications/launchpad/gui-react/src/store/index.ts

```diff
@@ -29,19 +29,33 @@ const persistConfig = {
   storage,
   blacklist: ['containers'],
 }
+
 const persistedReducer = persistReducer(persistConfig, reducer)
 
-export const store = configureStore({
-  reducer: persistedReducer,
-  enhancers: [
-    devToolsEnhancer({
-      name: 'Tari Launchpad',
-      // realtime: true, // enables devtools on production
-      hostname: 'localhost',
-      port: 8000,
-    }),
-  ],
-})
+export const store =
+  process.env.NODE_ENV === 'test'
+    ? configureStore({
+        reducer: persistedReducer,
+        enhancers: [
+          devToolsEnhancer({
+            name: 'Tari Launchpad',
+            // realtime: true, // enables devtools on production
+            hostname: 'localhost',
+            port: 8000,
+          }),
+        ],
+      })
+    : configureStore({
+        reducer: persistedReducer,
+        enhancers: [
+          devToolsEnhancer({
+            name: 'Tari Launchpad',
+            // realtime: true, // enables devtools on production
+            hostname: 'localhost',
+            port: 8000,
+          }),
+        ],
+      })
```

Comment:
Isn't both the same?

Reply:
OOPS

Reply:
fixed, test environment was supposed to use non-persisted reducer to avoid some unnecessary errors in the console